netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,186 @@
+= Upgrade to NetSuite 11.x - Mule 4
+:page-aliases: connectors::netsuite/netsuite-connector-upgrade-migrate.adoc
+
+Upgrade NetSuite Connector to version 11.x to use the new features.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|10.x |11.x
+|===
+
+== Changes in This Release
+
+In the v10.x versions of NetSuite Connector, the connector resolved metadata by receiving Java objects as input and converting them internally to XML so that NetSuite could understand them. In the 11.x version, NetSuite Connector resolves XML metadata directly.
+
+Because of this modification, the input and output metadata for connector operations has changed and is now equivalent to the request and response bodies of the corresponding SOAP API operation, respectively. You can use DataSense explorer in Studio to drag and drop the metadata fields, which causes their corresponding namespaces and tags to be generated automatically.
+
+Version 10.x also eliminates the REST-based operations.
+
+== Changes to Sources
+
+Removed the Page Size field for the sources (On Deleted Object, On Modified Object, and On New Object). Each source now returns one record in the output metadata.
+
+== Changes to Operations
+
+=== Advanced Operations
+
+Changes to advanced operations include the following:
+
+[%header%autowidth.spread]
+|===
+| Operation |Changes
+| Change Email | Returns a NetSuiteSoapResult object in the output metadata
+| Change Password | Returns a NetSuiteSoapResult object in the output metadata
+| Get Account Governance Info | Returns a GetAccountGovernanceInfoResult object in the output metadata
+| Get Data Center URLs | Returns a NetSuiteSoapResult object in the output metadata.
+| Get Server Time | Returns a NetSuiteSoapResult object in the output metadata
+|===
```

Comment:
Missing getIntegrationGovernanceInfo operation

Reply:
Does this return a GetIntegrationGovernanceInfoResult object?

Reply:
Yes